README.md

```diff
@@ -18,40 +18,51 @@ All of this results in a more iterative process for developing manifests, one th
 
 ## Install
 
-> You should have Go installed *(minimum version 1.8.1)*, and an appropriately set `$GOPATH`. If you need additional help, see the [official Go installation guide](https://golang.org/doc/install#install).
+> You should have Go installed *(minimum version 1.8.1)*. If not, follow the instructions in the [official installation guide](https://golang.org/doc/install#install).
 
-To install ksonnet, run the following commands:
+1. First ensure that you have a working `$GOPATH`:
+    ```
+    echo $GOPATH
+    ```
+    If your `$GOPATH` is empty, you'll need to permanently set it. On OSX, you can do this by adding the line `export GOPATH=$HOME/go` to the end of your `$HOME/.bash_profile`. (Otherwise [these instructions](https://github.com/golang/go/wiki/SettingGOPATH) may help).
 
-```bash
-# Download ksonnet
-go get github.com/ksonnet/ksonnet
 
-# Separate make command to install binary under shortname `ks`
-cd $GOPATH/src/github.com/ksonnet/ksonnet
-make install
+2. Now, to install ksonnet, run the following commands:
 
-# Ensure that Go binaries are runnable in any directory
-PATH=$PATH:$GOPATH/bin
-```
+    ```bash
+    # Make sure that your $GOPATH is set up! This is usually /usr/local/
+    # /usr
+    echo $GOPATH
 
-If your ksonnet is properly installed, you should be able to run the following `--help` command and see similar output:
+    # Download ksonnet
+    go get github.com/ksonnet/ksonnet
 
-```
-$ ks --help
+    # Separate make command to install binary under shortname `ks`
+    cd $GOPATH/src/github.com/ksonnet/ksonnet
+    make install
 
-Synchronise Kubernetes resources with config files
+    # Ensure that Go binaries are runnable in any directory
```

Comment:
I would move this up to step 1, so we can distinguish the ksonnet specific install from the go install.

Reply:
good call